nuxeo-importer-stream/src/main/java/org/nuxeo/importer/stream/consumer/BlobMessageConsumer.java

```diff
@@ -99,4 +122,37 @@ public void commit() {
     public void rollback() {
 
     }
+
+    public class MyBlob {
```

Comment:
Make the class `static`.
Could you rename it to `CloseableBlob` and implement `AutoCloseable` and use a `close` method instead of `clean`?